### PR TITLE
atuin: 15.0.0 -> 16.0.0

### PR DESCRIPTION
**Pull request by happysalada**
## Description of changes

https://github.com/atuinsh/atuin/releases

this mostly fixing a breakage with nushell 0.83 for me, so it would be amazing if this was merged quickly.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
> this mostly fixing a breakage with nushell 0.83 for me, so it would be amazing if this was merged quickly.

If you are in a hurry then overlays are always an option to get the update into your config immediately.

File: pkgs/tools/misc/atuin/default.nix [hunk 1]
```diff
@@ -11,20 +11,20 @@
...
   src = fetchFromGitHub {
-    owner = "ellie";
+    owner = "atuinsh";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-BX1WpvJMcfpepsRX0U6FJBL5/+mpUyTZxm65BbbZLJA=";
+    hash = "sha256-Kh6aaWYV+ZG7Asvw5JdGsV+nxD+xvvQab5wLIedcQcQ=";
   };
 
   # TODO: unify this to one hash because updater do not support this
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/atuin/default.nix:23]: No one has an idea how we can resolve this TODO?

**Review by SuperSandro2000 — COMMENTED** [review]
otherwise LGTM

File: pkgs/tools/misc/atuin/default.nix (hunk 1 above)
Comment (happysalada) [reply pkgs/tools/misc/atuin/default.nix:23]: no idea unfortunately

**Comment by happysalada** [comment]
@SuperSandro2000 took care of your comment.

**Review by yoav-lavi — APPROVED** [review]

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**